Modules/Microsoft365DSC/Modules/M365DSCUtil.psm1

```diff
@@ -1153,6 +1153,10 @@ Internal
 function Get-M365DSCTenantDomain
 {
     param(
+        [Parameter()]
+        [System.Management.Automation.PSCredential]
+        $Credential,
+
```

Comment:
as mentioned in https://github.com/microsoft/Microsoft365DSC/pull/1792#issuecomment-1061727808 this function should not handle any Credentials.